### PR TITLE
Front(fix): UI hot fixes [MARXAN-689]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Overview

- [x] Align Home

- [x] Change Carousel Home images

- [x] Change Blm chart images and set borders dinamically

- [x] Set back links on blue

- [x] Manage visibility on Solution sidebar

- [x] Delete double infoButton on features: 

- [x] Solutions Table improvements: Add title and set info button on its right side

- [x] Solutions Table improvements:  Colors: brighten blue and gray

- [x] Solutions Table improvements:  refactor data fetching cases

- [x] Solutions Table improvements: remove text of info button and add scroll to the whole modal

### Feature relevant tickets

[MARXAN-689](https://vizzuality.atlassian.net/secure/RapidBoard.jspa?rapidView=38&projectKey=MARXAN&modal=detail&selectedIssue=MARXAN-689&assignee=60d985f51613d50069adeabc)

### CODE REVIEW

File: app/layout/scenarios/show/solutions/table-form/component.tsx [hunk 1]
```diff
@@ -59,17 +59,12 @@ export const SolutionsTableForm: React.FC<SolutionsTableFormProps> = ({
     isFetched: mostDifSolutionsisFetched,
   } = useMostDifferentSolutions(sid);
 
-  const allSolutionsFetched = (!isFetching || isFetchingNextPage)
-  && data && data.length > 0 && !mostDifSolutions;
+  const allSolutionsFetched = solutionsAreFetched || isFetchingNextPage;
```
Comment (mbarrenechea) [inline app/layout/scenarios/show/solutions/table-form/component.tsx:62]:
```suggestion
  const allSolutionsFetched = solutionsAreFetched || mostDifSolutionsisFetched;
```

File: app/layout/scenarios/show/solutions/table-form/component.tsx [hunk 2]
```diff
@@ -59,17 +59,12 @@ export const SolutionsTableForm: React.FC<SolutionsTableFormProps> = ({
...
 
-  const mostDifSolutionsIsSelected = mostDifSolutions
-  && mostDifSolutionsData && mostDifSolutionsData.length > 0;
+  const noSolutionResults = (!solutionsAreFetching && !solutionsData.length)
```
Comment (mbarrenechea) [inline app/layout/scenarios/show/solutions/table-form/component.tsx:64]:
```suggestion
  const noSolutionResults = (solutionsAreFetched && !solutionsData.length)
```

File: app/layout/scenarios/show/solutions/table-form/component.tsx [hunk 3]
```diff
@@ -59,17 +59,12 @@ export const SolutionsTableForm: React.FC<SolutionsTableFormProps> = ({
...
+    || (!mostDifSolutionsisFetched && !mostDifSolutionsData.length);
```
Comment (mbarrenechea) [inline app/layout/scenarios/show/solutions/table-form/component.tsx:65]:
```suggestion
    || (mostDifSolutionsisFetched && !mostDifSolutionsData.length);
```

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

File: app/layout/scenarios/show/solutions/table-form/component.tsx (hunk 3 above)
Comment (anamontiaga) [reply app/layout/scenarios/show/solutions/table-form/component.tsx:65]: much better!

**Pull request merged**